pysilero/pickable_session.py

```
@@ -30,7 +30,12 @@ def __init__(self, version="v5"):
         opts.log_severity_level = 3
 
         assert version in ["v4", "v5"]
-        repo_dir = snapshot_download("pengzhendong/silero-vad")
+        try:
+            repo_dir = snapshot_download("pengzhendong/silero-vad")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)